src/main/java/Duke.java

```diff
@@ -1,10 +1,178 @@
+        inCommand = inCommand.trim();
+        String[] temp = inCommand.split(" ");
+        return temp;
+    }
+
+    public static void throwException() {
+        System.out.println(LINE);
+        System.out.println("\t Input command is wrong. Enter \"help\" for list of accepted");
+        System.out.println("\t commands");
+        System.out.println(LINE);
+    }
+
+    public static void listCommands() {
+        System.out.println(LINE);
+        System.out.println("\t "+ Arrays.toString(COMMAND));
+        System.out.println(LINE);
+    }
+
+    public static void executeBye() {
+        System.out.println(LINE);
+        System.out.println("\t Bye.Hope to see you again soon!");
+        System.out.println(LINE);
+    }
+
+    public static void executeList(ArrayList<Task>l1) {
+        if(l1.isEmpty()) {
+            System.out.println(LINE);
+            System.out.println("\t Oops! No task has been assigned yet! Please enter a task");
+            System.out.println("\t before listing");
+            System.out.println(LINE);
+            return;
+        }
+        int i = 0;
+        System.out.println(LINE);
+        System.out.print("\t Here are the tasks in your list:\n");
+        for (i = 0; i < l1.size(); i++) {
+            int count = i + 1;
+            Task temp = l1.get(i);
+            System.out.println("\t " + count +"."+ temp.getTaskDescription().trim());
+        }
+        System.out.println(LINE);
+    }
+
+    public static void executeDone(String userIn, ArrayList<Task> l1) {
+        String[] temp = getCommand(userIn);
+        if(temp.length>1) {
+            int number = Integer.parseInt(temp[1]) - 1;
+            if(number>=l1.size() || number<=0) {
+                System.out.println(LINE);
+                System.out.println("\t Task number provided is not valid. Press \"list\"to see");
+                System.out.println("\t the list of available task numbers");
+                System.out.println(LINE);
+                return;
+            }
+            Task tempTask = l1.get(number);
+            tempTask.done();
+            System.out.println(LINE);
+            System.out.println("\t Nice! I've marked this task as done:");
+            System.out.println("\t   "+tempTask.getTaskDescription());
+            System.out.println(LINE);
+        }else{
+            System.out.println(LINE);
+            System.out.println("\t Please enter the task number to be mark as done!");
+            System.out.println(LINE);
+        }
+    }
+
+    public static void updateTask(ArrayList<Task> l1, String userIn, String timing) {
+        String[] temp=getCommand(userIn);
+        String action="";
+        boolean flip=false;
+        for(int i=1; i<(temp.length); i++) {
+            if (temp[i].charAt(0) == '/') {
+                flip=true;
+            }
+            if(flip){
+                timing += temp[i] + " ";
+            }else{
+                action += temp[i] + " ";
```

Comment:
Shouldn't there be spaces before and after the equal sign? It is good to be consistent..